jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/ConnectorFactoryTest.java

```diff
@@ -38,7 +39,10 @@ public void ssl_jks_config_is_validated() {
                                              new Ssl.PemKeyStore.Builder()
                                                      .keyPath("nonEmpty"))));
 
-        ConnectorFactory willThrowException = new ConnectorFactory(config, new ThrowingSecretStore());
+        ThrowingSecretStore secretStore = new ThrowingSecretStore();
+        ConnectorFactory willThrowException = new ConnectorFactory(config,
```

Comment:
Please extract a utility method for creating the ConnectorFactory, as it's repeated in all the tests. Please consider if something is wrong with the design. It's an anti-pattern to send the same params to the ConnectorFactory and one of it's parameter objects.

Reply:
Will fix

Reply:
We could discuss changing the interface to take the `ConnectorConfig` as parameter to `configure()`. I would say that it is a coincidence that `DefaultSslKeyStoreConfigurator` also takes a `SecretStore`.